src/modules/site/base/forms/forms.py

```diff
@@ -192,16 +192,26 @@ def process_formdata(self, val):
     return super(FlexIntegerField, self).process_formdata(val)
 
 
+class StrainSelectField(SelectField):
+  def pre_validate(self, form):
+    pass
+
+
 class PairwiseIndelForm(Form):
-  STRAIN_CHOICES = get_indel_primer_strain_choices()
   CHROMOSOME_CHOICES = get_indel_primer_chrom_choices()
+  SPECIES_CHOICES = [(name, value.short_name) for name, value in SPECIES_LIST.items()]
   
-  strain_1 = SelectField('Strain 1', choices=STRAIN_CHOICES, default="N2", validators=[Required(), validate_uniq_strains])
-  strain_2 = SelectField('Strain 2', choices=STRAIN_CHOICES, default="CB4856", validators=[Required()])
+  species = SelectField('Species', choices=SPECIES_CHOICES, default="c_elegans", validators=[Required()])

```

Comment:
@vlagrassa can we move these default settings under the "Tool Version/Settings" in the admin interface? We can store these under datastore/etc. (goal is to to remove all hard-coded vars from code).